packages/api-browser/window.js

```diff
@@ -2,10 +2,20 @@ if (!window.ssf) {
   window.ssf = {};
 }
 
+window.accessableWindows = [];
```

Comment:
Yeah I think waiting for the rollup PR to go in is probably a good idea, because any other way we do it will just be changed to that as soon as its in anyway.